Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/bugzilla.py

```diff
@@ -159,6 +159,13 @@ def _issue(self, url, id, credentials=None, data=None):
             if data.get('version'):
                 issue['version'] = data['version']
 
+            keywords = data.get('keywords', {})
+            if keywords:
+                if keywords.get('set'):
+                    issue['keywords'] = keywords.get('set')
+                elif keywords.get('add'):
+                    issue['keywords'] = issue.get('keywords', []) + keywords.get('add')
```

Comment:
It looks like `set` has higher priority than `add`. I guess we only allow one key in `keywords` dictionary.
Maybe we should `assert len(keywords) == 1` and then logging / throwing error for unsupported keys.
In this way, we can check against malformed `keywords`.

Reply:
Want to note that this is actually a mock. That being said, an `assert` here would be nice.